packages/types/src/localization.ts

```diff
@@ -225,6 +236,11 @@ type _LocalizationResource = {
         title: LocalizationValue;
         primaryButton: LocalizationValue;
       };
+      backupCode2FaSection: {
```

Comment:
In some places we're using "mfa" in others "2fa". Let's align them. I think we should be using "mfa" everywhere